autoload/denops/_internal/plugin.vim

```diff
@@ -21,31 +21,38 @@ function! denops#_internal#plugin#load(name, script) abort
   const l:script = denops#_internal#path#norm(a:script)
   const l:args = [a:name, l:script]
   let l:plugin = denops#_internal#plugin#get(a:name)
-  let l:plugin.state = s:STATE_LOADING
+  if l:plugin.state ==# s:STATE_RESERVED || l:plugin.state ==# s:STATE_FAILED
+    let l:plugin.state = s:STATE_LOADING
+  endif
```

Comment:
Why do we just need a guard here? Don't we need to skip the rest of code when the `state` is not expected value?

Reply:
Since Deno handles multiple calls to `unload()`, I made the changes minimal.
It would certainly be better to enclose everything in if-endif, since there is a possibility that the values ​​in `plugin.script` may be inconsistent.

Reply:
The notifications to Deno should still not be guarded, since developers are free to call `load()` and will receive messages if debug mode is enabled.

Reply:
Changed only include `plugin.script` to if-endif.

Reply:
> The notifications to Deno should still not be guarded, since developers are free to call load() and will receive messages if debug mode is enabled.

I'm sorry but I don't get it. If the reason is just for debugging, I think only the message is required but notifications to Deno like:

```vim
  " ...
  if l:plugin.state !=# s:STATE_RESERVED && l:plugin.state !=# s:STATE_FAILED
    call denops#_internal#echo#debug(printf('already loaded. skip: %s', l:args))
    return
  endif

  let l:plugin.state = s:STATE_LOADING
  let l:plugin.script = l:script
  call denops#_internal#echo#debug(printf('load plugin: %s', l:args))
  call denops#_internal#server#chan#notify('invoke', ['load', l:args])
endfunction
```

Reply:
I did so.